index.bs

```diff
@@ -1386,6 +1402,20 @@ Each {{XRView}} has an associated <dfn for="XRView">internal projection matrix</
 
 Note: The {{XRView/transform}} can be used to position camera objects in many rendering libraries. If a more traditional view matrix is needed by the application one can be retrieved by calling `view.transform.inverse.matrix`.
 
+<div class="algorithm" data-algorithm="request-viewport-scale">
+
+The <dfn method for="XRView">requestViewportScale(|scale|)</dfn> method requests that the user agent should set the [=view/requested viewport scale=] for this viewport to the requested value.
+
+When this method is invoked on a [=view=] |view|, the user agent MUST run the following steps:
```

Comment:
The method cannot be invoked on a `[=view=]` (an abstract concept that is per-session), rather it is invoked on an `{{XRView}}`, which contains a reference to a `[=view=]`

Reply:
Done, thank you for the clarification.